content/20.treatments.md

```diff
@@ -11,48 +11,91 @@ In the following sections, we critically appraise the literature surrounding the
+This indicates an inflammatory response out of control or what is called “cytokine storm”.
+In this context IL-6 plays a key role [@doi:10.1128/mmbr.05015-11].
+IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation.
+Interleukins can regulate both pro- and anti-inflammatory response.
+In this context, severe lung infection reflects the over-production of proinflammatory interleukins. 
+In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.
+It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽. These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
+The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
+Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus. 
+Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, may palliate the most severe symptoms associated with COVID-19. 
 
 ##### Anticipated Mechanism 
 
-Why it may be useful
+Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. 
 
 ##### Current Evidence
 
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results. 
+Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. 
+Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. 
+The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).
```

Comment:
Maybe for these upcoming studies, the way to go is perhaps to provide a link at the end of the current evidence section since the data doesn't exist yet?